humanism.sh

```diff
@@ -6,7 +6,13 @@
 #
 # Common aliases
 #
-shopt -s expand_aliases
+
+SH_SOURCE=${BASH_SOURCE:-$_}
+
+if ! shopt -s expand_aliases >/dev/null 2>&1; then
```

Comment:
Not sure if we want to lose output here, but I thought it was not very important, and it removes annoying warning for non-bash users